tests/shell/tests.py

```diff
@@ -17,3 +19,13 @@ def test_command_option(self):
             )
             self.assertEqual(len(logger), 1)
             self.assertEqual(logger[0], __version__)
+
+    @mock.patch('django.core.management.commands.shell.select')
```

Comment:
these aren't necessary, you can pass `StringIO`s for `stdout` and `stdin` to `call_command` instead. there are lots of examples in existing tests e.g. https://github.com/django/django/blob/master/tests/i18n/test_compilation.py#L41